collector/tcpstat.go

```diff
@@ -0,0 +1,210 @@
+// +build !notcpstat
+
+package collector
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	procTcpStat  = "/proc/net/tcp"
+	procTcp6Stat = "/proc/net/tcp6"
+)
+
+const (
```

Comment:
(corrected some typos in the above comment, but notification mails are only sent once, so just calling it out)